lib/readline.js

```diff
@@ -126,7 +133,7 @@ function Interface(input, output, completer, terminal) {
   this.removeHistoryDuplicates = !!removeHistoryDuplicates;
   this.crlfDelay = crlfDelay ?
     Math.max(kMincrlfDelay, crlfDelay) : kMincrlfDelay;
-
+  this.escapeCodeTimeout = escapeCodeTimeout;
```

Comment:
`escapeCodeTimeout` isn't used between here and line 101. I don't think you need to create an intermediate variable for it.